CHANGELOG.md

```diff
@@ -180,6 +177,9 @@ Adding a new version? You'll need three changes:
 - Support `ExtensionRef` HTTPRoute filter. It is now possibile to set a KongPlugin
   reference in the `HTTPRoute`s' `ExtensionRef` filter field.
   [#4838](https://github.com/Kong/kubernetes-ingress-controller/pull/4838)
+- Added `--kong-admin-token-file` flag to provide the Kong admin token via file
```

Comment:
1. I'd add a note here that this is mutually exclusive with `--kong-admin-token`.
2. Could we also say in this flag's description that it's mutually exclusive with `--kong-admin-token`? This way users don't have to actually run KIC to figure this out in the [config validation ](https://github.com/Kong/kubernetes-ingress-controller/blob/82dbd35055574cd7c438a29c5235f965c4592fe1/internal/manager/config_validation.go#L64-L66)